src/webnn_native/GraphBuilder.cpp

```diff
@@ -140,8 +140,10 @@ namespace webnn_native {
         if (GetContext()->ConsumedError(graph->Finish())) {
             BUILD_ERROR_AND_CALLBACK("Failed to finish building graph.");
         }
-        callback(MLBuildStatus_Success, reinterpret_cast<MLGraph>(graph.Detach()), nullptr,
-                 userdata);
+        // Compile the graph, call back the graph to user if sucessful, otherwise delete it in async
+        // thread like
+        // https://github.com/otcshare/webnn-native/commit/c16976d063b5dbbe9de85cd12a979814423fd232.
```

Comment:
Please show the code here, otcshare/webnn-native is a private repo.

Reply:
Remove the private code.